msdk-id-sirius/src/main/java/ISirius.java

```diff
@@ -0,0 +1,73 @@
+import de.unijena.bioinf.ChemistryBase.chem.Ionization;
+import de.unijena.bioinf.ChemistryBase.ms.Ms2Experiment;
+import de.unijena.bioinf.ChemistryBase.ms.Peak;
+import de.unijena.bioinf.ChemistryBase.ms.Spectrum;
+import de.unijena.bioinf.sirius.IdentificationResult;
+import de.unijena.bioinf.sirius.Sirius;
+import javafx.util.Pair;
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+/**
+ * Created by evger on 14-May-18.
+ */
+
+public class ISirius {
+
+    public static Pair<double[], double[]> readMsFile(String path) throws FileNotFoundException {
+        List<Double> mz, intensive;
+        mz = new ArrayList<>();
+        intensive = new ArrayList<>();
+        Scanner sc = new Scanner(new File(path));
+        String words[];
+
+        while (sc.hasNext()) {
+//            words = sc.nextLine().split(" ")
+            mz.add(sc.nextDouble());
+            intensive.add(sc.nextDouble());
+        }
+
+        sc.close();
+
+        Pair<double[], double[]> pair;
+        Double mzArray[] = mz.toArray(new Double[mz.size()]);
+        Double intensiveArray[] = intensive.toArray(new Double[intensive.size()]);
+
+        double[] mzPrimitive = Stream.of(mzArray).mapToDouble(Double::doubleValue).toArray();
+        double[] intensivePrimitive = Stream.of(intensiveArray).mapToDouble(Double::doubleValue).toArray();
+
+        return new Pair<>(mzPrimitive, intensivePrimitive);
+    }
+
+    public static void main(String[] args) {
```

Comment:
There should be no main() function here. Instead, you can start writing simple junit tests in the src/test/java folder.
If you have no experience with junit, read some tutorials first.